query/common/src/main/java/org/codice/ddf/admin/common/fields/common/MapField.java

```diff
@@ -47,4 +46,8 @@ public MapField put(String key, String value) {
         return this;
     }
 
+    @Override
+    public void initializeFields() {
+        entries = new ListFieldImpl<>("entries", PairField.class);
```

Comment:
Can you leave a todo here to replace with a MapField? I have it implemented but not pushed.